front/app/components/admin/PostManager/components/PostPreview/Initiative/AdminInitiativeEdit.tsx

```diff
@@ -105,7 +107,14 @@ const AdminInitiativeEdit = ({ goBack, initiativeId }: Props) => {
           </Box>
         </Box>
       </Top>
-      <Content>
+      <Box
+        py="30px"
+        pl="35px"
+        pr="35px"
+        mt="0"
+        width="100%"
+        background={colors.background}
```

Comment:
Does the same as the imported Content component did, but needed to add a custom style.